HarfangLab/harfanglab/ingest/parser.yml

```
@@ -978,3 +992,26 @@ stages:
           process.executable: "{{json_event.message.process_image}}"
           process.pid: "{{json_event.message.process_id}}"
           process.thread.id: "{{json_event.message.thread_id}}"
+
+  full_agent_info:
+    actions:
+      - set:
+          event.start: "{{parse_first_seen.datetime}}"
+          event.end: "{{parse_last_seen.datetime}}"
+          event.category: ["agent"]
```

The `agent` category doesn't exist. Please change to `host`.
```suggestion
          event.category: ["host"]
```